tests/integration/test_vulnerability_detector/test_feeds/test_cpe_indexing.py

```diff
@@ -0,0 +1,280 @@
+
+tags:
+    - vulnerability_detector
+    - cpe_helper
+'''
+import os
+import pytest
+
+from wazuh_testing.constants.paths.logs import WAZUH_LOG_PATH
+from wazuh_testing.utils.configuration import (load_configuration_template, get_test_cases_data,
+                                               update_configuration_template)
+from wazuh_testing.utils.db_queries import agent_db
+from wazuh_testing.utils.callbacks import generate_callback
+from wazuh_testing.utils.file import read_json_file, copy, write_json_file
+from wazuh_testing.tools.file_monitor import FileMonitor
+from wazuh_testing.modules.vulnerability_detector import patterns as cb
+from wazuh_testing.modules.vulnerability_detector import (CUSTOM_NVD_FEED, CUSTOM_CPE_HELPER, CPE_HELPER_PATH,
+                                                          CUSTOM_MSU_JSON_FEED)
+from wazuh_testing.modules.vulnerability_detector.packages import VULNERABLE_PACKAGES
+from wazuh_testing.modules.vulnerability_detector import LOCAL_INTERNAL_OPTIONS as local_internal_options
+
+from . import (TEST_CASES_PATH, CONFIGURATIONS_PATH, TEST_FEEDS_PATH, custom_nvd_json_feed_path,
+               custom_cpe_helper_path, custom_msu_feed_path)
+
+pytestmark = [pytest.mark.server]
+
+# Configuration and cases data
+configurations_path = os.path.join(CONFIGURATIONS_PATH, 'configuration_cpe_indexing.yaml')
+t1_cases_path = os.path.join(TEST_CASES_PATH, 'cases_cpe_indexing_packages.yaml')
+t2_cases_path = os.path.join(TEST_CASES_PATH, 'cases_cpe_indexing_system.yaml')
+
+# CPE indexing packages test configurations (t1)
+t1_configuration_parameters, t1_configuration_metadata, t1_test_case_ids = get_test_cases_data(t1_cases_path)
+t1_configurations = load_configuration_template(configurations_path, t1_configuration_parameters,
+                                                t1_configuration_metadata)
+t1_systems = [metadata['system'] for metadata in t1_configuration_metadata]
+
+# CPE indexing system test configurations (t2)
+t2_configuration_parameters, t2_configuration_metadata, t2_test_case_ids = get_test_cases_data(t2_cases_path)
+t2_configurations = load_configuration_template(configurations_path, t2_configuration_parameters,
+                                                t2_configuration_metadata)
+t2_systems = [metadata['system'] for metadata in t2_configuration_metadata]
+
+# Set offline custom feeds configuration
+t1_configurations = update_configuration_template(t1_configurations, ['CUSTOM_NVD_JSON_PATH'],
+                                                  [custom_nvd_json_feed_path])
+t2_configurations = update_configuration_template(t2_configurations, ['CUSTOM_NVD_JSON_PATH', 'CUSTOM_MSU_JSON_PATH'],
+                                                  [custom_nvd_json_feed_path, custom_msu_feed_path])
+# Global vars
+agent_packages = VULNERABLE_PACKAGES
+
+
+@pytest.fixture()
+def prepare_scan(agent_system, mock_agent_with_custom_system):
```

Comment:
This has to be in a `conftest.py` file, not here. Apply this to all fixtures located in test files.

Reply:
This is a fixture specific for this test, no reason to move it to conftest.py

Reply:
done in https://github.com/wazuh/wazuh/pull/17576/commits/3749243c366e3a978305ce04345854fcd7d62079